multi/submitblindedproposal.go

```diff
@@ -0,0 +1,35 @@
+// Copyright © 2021, 2025 Attestant Limited.
```

Comment:
Please update the copyright. 
```suggestion
// Copyright © 2025 Attestant Limited.
```